src/components/markdown/markdown.scss

```diff
@@ -30,7 +30,7 @@
 .page-h2,
 .component-docs h2,
 .icon-h2 {
-  //@include type-style('expressive-heading-04', true);
+  //@include carbon--type-style('expressive-heading-04', true);
```

Comment:
It this commented out intentionally?

Reply:
I dunno, I just did a find and replace. Looks like this was already commented out. I can just remove it entirely.